extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -41,6 +41,8 @@ tags: {  # <amp-list>
     blacklisted_value_regex: "__amp_source_origin"
   }
   attrs: { name: "items" }
+  attrs: { name: "single-item" }
```

Comment:
nit: alphabetize the non-bind attributes. also noticed `items` wasn't in previous version, please alphabetize that too. thx!